README.md

```diff
@@ -101,11 +109,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
```

Comment:
`.innerHTML` needs to be called on a single DOM object. `getElementsByClassName` returns a collection of them, so you need to isolate which one you want the `.innerHTML` of, hence the brackets.